packages/core/scss/global/elements/_elements.details.scss

```diff
@@ -34,6 +34,13 @@ details {
       font-family: $font;
     }
   }
+
+  table {
+    position: static !important;
```

Comment:
Dette vil vel gjelde for alle tabeller. Gammel kode targeta jo tabeller i details. Vil det være problematisk?

Reply:
Denne skal kun targete tabeller i details?  Den ligger 
`
details {
     table {...}
}     
`

Reply:
Ah. La ikkje merke til det.